### DISCUSSION TITLE
How to disable all engines but open only one?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [searxng/searxng#3064] [Q&A] [answered]
As you know, in some place on earth, internet is very strict. 

google, wikipedia, wolfpha, stckoverflow.... all can not access in this place.

How to disable all of them but just open bing?

**Comment by unixfox** [comment]
Next time please use github discussions. This is not a bug or a feature request.

**Reply by lucasjinreal** [reply]
this is a feature request

**Reply by unixfox** [reply]
No it's not. This feature already exist.

**Reply by lucasjinreal** [reply]
how

**Comment by return42** [comment, marked as answer]
> How to disable all engines but open only one? 

@lucasjinreal you can `disable` or `inactive` the engines you don't want --> https://docs.searxng.org/admin/settings/settings_engine.html#settings-engine .. 

> this is a feature request

Really? .. For me it was a question, but if it is a request, the answer will be:

An option to disabling all engines except one does not fit into a concept of a metasearch engine.
